bower.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "purescript-halogen-datapicker",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/slamdata/purescript-halogen-datapicker.git"
+  },
+  "authors": [
+    "Irakli Safareli <i.safareli@gmail.com>"
+  ],
+  "ignore": [
+    "**/.*",
+    "bower_components",
+    "node_modules",
+    "output",
+    "tests",
+    "tmp",
+    "bower.json",
+    "package.json",
+    "example"
+  ],
+  "dependencies": {
+    "purescript-enums": "^3.2.0",
+    "purescript-formatters": "git://github.com/safareli/purescript-formatters.git#interval",
```

Comment:
purescript-formatters must be merged before this pr, so this will be changed to just version.